src/slack/SlackClient.ts

```diff
@@ -15,6 +20,34 @@ export class SlackClient {
     this.channel = channel
   }
 
+  /**
+   * Return the set of non-bot users.
+   *
+   * @returns `null` if the bot token is missing the required OAuth scope
+   */
+  async getRealUsers(): Promise<Member[] | null> {
+    try {
+      const {members} = await this.web.users.list()
```

Comment:
[This API method](https://api.slack.com/methods/users.list) falls under [Tier 2](https://api.slack.com/docs/rate-limits#tier_t2) rate limiting:

> Most methods allow at least 20 requests per minute, while allowing for occasional bursts of more requests.

The Node.js Slack SDK [retries such errors](https://slack.dev/node-slack-sdk/web-api#rate-limits) by default, which seems useful at the expense of increasing the GitHub workflow step duration. Probably something to keep an eye on.